tests/lib/prepare-restore.sh

```diff
@@ -132,6 +133,13 @@ build_rpm() {
     done
     distro_install_package "${deps[@]}"
 
+    # TODO: remove this
+    # Force the version file generation in centos-9, This is needed because when
+    # the rpm is built, the file version_generated.go is not used to build the binary.
+    if os.query is-centos-9; then
+        ./mkversion.sh "$version-$release"
+    fi
```

Comment:
I don't think we should do this here but rather try to work out why things are not working as expected. I try to keep the snapd spec close to the downstream one in Fedora. Keeping the two in sync gives me some confidence in the packages we release to the distro.